src/components/SideBar.jsx

```diff
@@ -49,11 +66,20 @@ const SideBar = ({ collections }) => {
               Collections
             </Typography>
             <div>
-              {collections.map((collections) => {
+              {collections.map((collection) => {
                 return (
-                  <ListItem key={collections.name}>
+                  <ListItem
+                    key={collection.uuid}
+                    secondaryAction={
+                      <IconButton
+                        onClick={() => handleDeleteCollection(collection.uuid)}
+                      >
+                        <DeleteTwoToneIcon />
```

Comment:
Users shouldn't be able to delete the Default collection, as the backend makes the assumption that one always exists. I think you could just make rendering the trash can IconButton conditional.

<img width="268" alt="Screenshot 2023-12-02 at 3 01 43 PM" src="https://github.com/secure-password-manager/PasswordManagerWeb/assets/39425112/c6e47d6b-84e1-47e7-a310-cbf1d0a0eedd">

Reply:
Yes good call, this has been updated in next commit